nixnet/_cconsts.py

```diff
@@ -105,7 +105,6 @@
 NX_ERR_FRAME_SIZE_MISMATCH = (NX_ERROR_BASE | 0x0AA)
 NX_ERR_INDEX_TOO_BIG = (NX_ERROR_BASE | 0x0AB)
 NX_ERR_SESSION_MODE_INCOMPATIBILITY = (NX_ERROR_BASE | 0x0AC)
-NX_ERR_SESSION_TYPE_FRAME_INCOMPATIBILITY = (NX_ERROR_BASE | 0x15D)
```

Comment:
These were moved and not removed

Reply:
Yup, a change to nixnet.h sorted them (by error code) and so I did too, as mentioned in the commit.